hack/patches/001-kourier-deployment.patch

```diff
@@ -0,0 +1,29 @@
+diff --git a/vendor/knative.dev/operator/pkg/reconciler/knativeserving/ingress/kourier.go b/vendor/knative.dev/operator/pkg/reconciler/knativeserving/ingress/kourier.go
+index 249847a9..4129aa33 100644
+--- a/vendor/knative.dev/operator/pkg/reconciler/knativeserving/ingress/kourier.go
++++ b/vendor/knative.dev/operator/pkg/reconciler/knativeserving/ingress/kourier.go
+@@ -38,13 +38,13 @@ var kourierFilter = ingressFilter("kourier")
+ 
+ func kourierTransformers(ctx context.Context, instance *v1alpha1.KnativeServing) []mf.Transformer {
```

Comment:
I think patching could be avoided if we move extension transformers at the very end [here](https://github.com/knative/operator/blob/0b56a26eb852d46c35a4ac0e677050afa9b33e14/pkg/reconciler/knativeserving/knativeserving.go#L141) (initially goal was not to change knative manifests only add platform specific).